code/_onclick/item_attack.dm

```diff
@@ -57,136 +54,92 @@
 			else
 				return 1
 
-	if(istype(M,/mob/living/carbon/brain))
-		var/mob/living/carbon/brain/B = M
-		messagesource = B.container
-	if(hitsound && force > 0)
-		playsound(loc, hitsound, 50, 1, -1)
-	/////////////////////////
+	if(!force)
+		playsound(loc, 'sound/weapons/tap.ogg', get_clamped_volume(), 1, -1)
+	else if(hitsound)
+		playsound(loc, hitsound, get_clamped_volume(), 1, -1)
+
 	user.lastattacked = M
 	M.lastattacker = user
-	add_logs(user, M, "attacked", name, "(INTENT: [uppertext(user.a_intent)]) (DAMTYE: [uppertext(damtype)])", print_attack_log = (force > 0))//print it if stuff deals damage
 
 	if(!iscarbon(user))
 		M.LAssailant = null
 	else
 		M.LAssailant = user
 
-	/////////////////////////
-
-	if(isanimal(M))
```

Comment:
What follows below is probably the most egregious example of OOP violation in the entire codebase.